_schema.graphql

```diff
@@ -48,17 +48,10 @@ type AggregationCount {
 type AnalyticsArtworksPublishedStats {
   percentageChanged: Int!
   period: AnalyticsQueryPeriodEnum!
-  timeSeries: [AnalyticsArtworksPublishedTimeSeriesStats!]
+  timeSeries: [AnalyticsPartnerTimeSeriesStats!]!
```

Comment:
Once metaphysics is deployed to production with this change, the following [Force](https://github.com/artsy/force/blob/13bb4efd42e0c872beefed7c4406264fda682521/.circleci/config.yml#L16-L23) and [Volt](https://github.com/artsy/volt/blob/41ab4719e339930911e8a23d032e7525d1d4186a/.circleci/config.yml#L35-L42) production deploy might fail due to breaking schema changes. We might want to update the MP schema in reaction, volt, (and emission?).

![Screen Shot 2019-05-16 at 2 14 33 PM](https://user-images.githubusercontent.com/796573/57877106-fe0ca080-77e4-11e9-961a-da8a09fe2645.png)

![Screen Shot 2019-05-16 at 2 14 02 PM](https://user-images.githubusercontent.com/796573/57877107-fe0ca080-77e4-11e9-889c-74f2bf6f3e52.png)